aws/src/main/java/org/apache/iceberg/aws/AwsProperties.java

```diff
@@ -103,6 +103,12 @@ public class AwsProperties implements Serializable {
   public static final String GLUE_CATALOG_SKIP_ARCHIVE = "glue.skip-archive";
   public static final boolean GLUE_CATALOG_SKIP_ARCHIVE_DEFAULT = false;
 
+  /**
+   * If Glue should skip name validations
```

Comment:
add some more explanations here, it is recommended to stick to Glue best practice in https://docs.aws.amazon.com/athena/latest/ug/glue-best-practices.html to make sure operations are Hive compatible. This is only added for services that have existing conventions using non-standard characters. When database name and table name validation is skipped, there is no guarantee that downstream systems would all support the names.